src/test/java/org/breedinginsight/brapi/v2/BrAPIV2ControllerIntegrationTest.java

```diff
@@ -0,0 +1,391 @@
+
+        BrAPIObservationVariable fetchedVariable = brAPIObservationVariableSingleResponse.getResult();
+        assertNotNull(fetchedVariable, "Observation Variable was not found");
+        assertEquals(createdVariable.getObservationVariableDbId(), fetchedVariable.getObservationVariableDbId());
+        //make sure the original values sent in the POST were saved correctly
+        assertEquals(variable.getObservationVariableName(), fetchedVariable.getObservationVariableName());
+        assertEquals(variable.getExternalReferences(), fetchedVariable.getExternalReferences());
+        assertEquals(variable.getCommonCropName(), fetchedVariable.getCommonCropName());
+
+        assertNotNull(fetchedVariable.getTrait(), "Trait is null");
+        assertEquals(createdVariable.getTrait().getTraitDbId(), fetchedVariable.getTrait().getTraitDbId());
+        //make sure the original values sent in the POST were saved correctly
+        assertEquals(variable.getTrait().getTraitName(), fetchedVariable.getTrait().getTraitName());
+        assertEquals(variable.getTrait().getTraitClass(), fetchedVariable.getTrait().getTraitClass());
+
+
+        assertNotNull(fetchedVariable.getMethod(), "Method is null");
+        assertEquals(createdVariable.getMethod().getMethodDbId(), fetchedVariable.getMethod().getMethodDbId());
+        //make sure the original values sent in the POST were saved correctly
+        assertEquals(variable.getMethod().getMethodName(), fetchedVariable.getMethod().getMethodName());
+        assertEquals(variable.getMethod().getMethodClass(), fetchedVariable.getMethod().getMethodClass());
+
+        assertNotNull(fetchedVariable.getScale(), "Scale is null");
+        assertEquals(createdVariable.getScale().getScaleDbId(), fetchedVariable.getScale().getScaleDbId());
+        //make sure the original values sent in the POST were saved correctly
+        assertEquals(variable.getScale().getScaleName(), fetchedVariable.getScale().getScaleName());
+
+        Flowable<HttpResponse<String>> getScaleCall = biClient.exchange(
+                GET(String.format("%s/programs/%s/brapi/v2/scales/%s",
+                                  biApiVersion,
+                                  validProgram.getId().toString(),
+                                  createdVariable.getScale().getScaleDbId()))
+                        .bearerAuth("test-registered-user"), String.class
+        );
+
+        HttpResponse<String> getScaleResponse;
+        try {
+            getScaleResponse = getScaleCall.blockingFirst();
+        } catch (Exception e) {
+            throw new Exception(e);
+        }
+        assertEquals(HttpStatus.OK, getScaleResponse.getStatus());
+
+        BrAPIScale scaleResponse = GSON.fromJson(JsonParser.parseString(getScaleResponse.body()).getAsJsonObject().getAsJsonObject("result"), BrAPIScale.class);
+
+        //TODO this is not being returned
```

Comment:
Is this a brapi server problem? Not a call we've been using

Reply:
Yeah, was a problem with the BrAPI server.  I'll create an issue on that repo.